pkg/k8s/client.go

```diff
@@ -74,6 +76,7 @@ func init() { K8ClientService = &k8ClientService{} }
 // obj must be a struct pointer so that obj can be updated with the response
 // returned by the Server.
 func (k8Client k8ClientService) Get(ctx context.Context, key client.ObjectKey, obj client.Object) error {
+	Client().Scheme()
```

Comment:
Why calling `Client().Scheme()` ?

Reply:
should be removed, it's a typing mistake